editor/components/page-attributes/order.js

```diff
@@ -28,24 +28,33 @@ export function PageAttributesOrder( { onUpdateOrder, instanceId, order } ) {
 	const inputId = `editor-page-attributes__order-${ instanceId }`;
 
 	return (
-		<PageAttributesCheck>
-			<label htmlFor={ inputId }>
+		<div>
+			<label key="label" htmlFor={ inputId }>
 				{ __( 'Order' ) }
 			</label>
-			<input
+			<input key="input"
 				type="text"
 				value={ order || 0 }
 				onChange={ setUpdatedOrder }
 				id={ inputId }
 				size={ 6 }
 			/>
-		</PageAttributesCheck>
+		</div>
+	);
+}
+
+function PageAttributesOrderWithChecks( props ) {
```

Comment:
Aside: Not usually a fan of multiple components defined in a single file, and this particular use makes me curious if an HoC pattern for checks could be simpler to integrate into our `compose` convention:

```js
export default compose( [
	ifPostTypeSupports( [ 'page-attributes' ] ),
] )( PageAttributesOrder );
```

Reply:
I agree with the suggestion, as future work an HoC will be implemented.